blas/unit_test/Test_Blas2_ger.hpp

```diff
@@ -0,0 +1,1538 @@
+  KOKKOS_IMPL_DO_NOT_USE_PRINTF(
+      "+======================================================================="
+      "===\n");
+  KOKKOS_IMPL_DO_NOT_USE_PRINTF("Starting %s, device = %s ...\n",
+                                caseName.c_str(), typeid(Device).name());
+
+#if defined(KOKKOSKERNELS_INST_LAYOUTLEFT) || \
+    (!defined(KOKKOSKERNELS_ETI_ONLY) &&      \
+     !defined(KOKKOSKERNELS_IMPL_CHECK_ETI_CALLS))
+  KOKKOS_IMPL_DO_NOT_USE_PRINTF(
+      "+-----------------------------------------------------------------------"
+      "---\n");
+  KOKKOS_IMPL_DO_NOT_USE_PRINTF("Starting %s for LAYOUTLEFT ...\n",
+                                caseName.c_str());
+
+  if (true) {
+    Test::GerTester<ScalarX, Kokkos::LayoutLeft, ScalarY, Kokkos::LayoutLeft,
+                    ScalarA, Kokkos::LayoutLeft, Device>
+        tester;
+    tester.test(0, 13, 0);
+    tester.test(1024, 0, 0);
+    tester.test(1, 1, 0);
+    tester.test(2, 2, 0);
+    tester.test(1, 2, 0);
+    tester.test(13, 13, 0);
+    tester.test(13, 1024, 0);
+    tester.test(13, 1024, 0, true, false);
+    tester.test(13, 1024, 0, true, true);
+    tester.test(50, 40, 4);
+    tester.test(1024, 1024, 0);
+    tester.test(2131, 2131, 0);
+    tester.test(2131, 2131, 0, true, false);
+    tester.test(2131, 2131, 0, true, true);
+  }
+
+  KOKKOS_IMPL_DO_NOT_USE_PRINTF("Finished %s for LAYOUTLEFT\n",
+                                caseName.c_str());
+  KOKKOS_IMPL_DO_NOT_USE_PRINTF(
+      "+-----------------------------------------------------------------------"
+      "---\n");
+#endif
+
+#if defined(KOKKOSKERNELS_INST_LAYOUTRIGHT) || \
+    (!defined(KOKKOSKERNELS_ETI_ONLY) &&       \
+     !defined(KOKKOSKERNELS_IMPL_CHECK_ETI_CALLS))
+  KOKKOS_IMPL_DO_NOT_USE_PRINTF(
+      "+-----------------------------------------------------------------------"
+      "---\n");
+  KOKKOS_IMPL_DO_NOT_USE_PRINTF("Starting %s for LAYOUTRIGHT ...\n",
+                                caseName.c_str());
+
+  if (true) {
+    Test::GerTester<ScalarX, Kokkos::LayoutRight, ScalarY, Kokkos::LayoutRight,
+                    ScalarA, Kokkos::LayoutRight, Device>
+        tester;
+    tester.test(0, 13, 0);
+    tester.test(1024, 0, 0);
+    tester.test(1, 1, 0);
+    tester.test(2, 2, 0);
+    tester.test(1, 2, 0);
+    tester.test(13, 13, 0);
+    tester.test(13, 1024, 0);
+    tester.test(13, 1024, 0, true, false);
+    tester.test(13, 1024, 0, true, true);
+    tester.test(50, 40, 4);
+    tester.test(1024, 1024, 0);
+    tester.test(2131, 2131, 0);
+    tester.test(2131, 2131, 0, true, false);
+    tester.test(2131, 2131, 0, true, true);
+  }
+
+  KOKKOS_IMPL_DO_NOT_USE_PRINTF("Finished %s for LAYOUTRIGHT\n",
+                                caseName.c_str());
+  KOKKOS_IMPL_DO_NOT_USE_PRINTF(
+      "+-----------------------------------------------------------------------"
+      "---\n");
+#endif
+
+#if 0  // Compilation error "static assertion failed Layout is not constructible
```

Comment:
You should be able to re-enable this if you follow the work by Brian in PR #1794

Reply:
Thanks for the hint. Done.